tasks/coverage/snapshots/semantic_typescript.snap

```diff
@@ -20478,6 +20478,14 @@ Scope children mismatch:
 after transform: ScopeId(0): [ScopeId(1)]
 rebuilt        : ScopeId(0): []
 
+semantic Error: tasks/coverage/typescript/tests/cases/compiler/moduleAndInterfaceSharingName2.ts
```

Comment:
I'm not sure how to understand these diffs.

Should I apply error code filtering only to the `parser_typescript' flow?